ide/static/js/content.js

```diff
@@ -755,7 +756,7 @@ class Content extends React.Component {
         },
         success : function (response) {
           if (response.result == 'success'){
-            var url = 'http://fabrik.cloudcv.org/caffe/load?id='+response.id;
+            var url = 'http://localhost:8000/caffe/load?id='+response.id;
             this.modalHeader = 'Your model url is:';
```

Comment:
For testing purpose I have made it localhost right now, when moving to prod I'll update this.